src/query/src/sql.rs

```diff
@@ -452,6 +497,25 @@ fn parse_file_table_format(options: &HashMap<String, String>) -> Result<Box<dyn
     )
 }
 
+async fn get_table_types(
+    tables: &Vec<String>,
+    catalog_manager: CatalogManagerRef,
+    query_ctx: QueryContextRef,
+    schema_name: &str,
+) -> Result<Arc<dyn Vector>> {
+    let mut table_types: Vec<String> = Vec::with_capacity(tables.len());
```

Comment:
Can the compiler infer the type?
```suggestion
    let mut table_types = Vec::with_capacity(tables.len());
```

Reply:
Yeahh, it can